libraries/AP_Scripting/docs/docs.lua

```diff
@@ -597,6 +597,20 @@ function Parameter_ud:init_by_info(key, group_element, type) end
 function Parameter_ud:init(name) end
 
 
+-- desc
+---@class rtc
+rtc = {}
+
+-- desc
```

Comment:
You have a nice description in the example, might as well add it here too. 
```suggestion
-- fills in year, month, day, hour, min, sec and ms
-- year is the regular Gregorian year, month is 0~11, day is 1~31, hour is 0~23, minute is 0~59, second is 0~60 (1 leap second), ms is 0~999
-- values are nil if there is no valid time source 
```